dolphinscheduler-ui/src/js/conf/home/pages/datasource/pages/list/_source/list.vue

```diff
@@ -32,6 +32,7 @@
             </div>
           </template>
         </el-table-column>
+        <el-table-column prop="note" :label="$t('Description')"></el-table-column>
```

Comment:
<el-table-column :label="$t('Description')" min-width="100">
          <template slot-scope="scope">
            <span>{{scope.row.note | filterNull}}</span>
          </template>
        </el-table-column>

Reply:
ok,I have corrected and checked again